scripts/validate-accounts-hook.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# Pre-commit hook to validate account definitions
```

Comment:
if the idea is not have this check as part of pre-commit hook, remove this line

Reply:
I updated the code to have this as a pre-commit hook along with tests in workflow.